src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
+import React from 'react';
 // export function Sum() {}
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  );
+}
```

Comment:
Also, i recommend you use the function expression
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
  );
```